src/Stryker.Core/Stryker.Core/Reporters/Progress/ProgressBarReporter.cs

```diff
@@ -17,12 +17,13 @@ public class ProgressBarReporter : IProgressBarReporter
         private const char ProgressBarDoneToken = '\u2588';
         private const char ProgressBarLeftToken = '-';
         private const int ProgressBarInitialState = -1;
-        private const string LoggingFormat = "Tests progress | {0} | {1} / {2} | {3} % |";
+        private const string LoggingFormat = "Tests progress | {0} | {1} / {2} | {3} % | rem time: {4} |";
```

Comment:
I prefer the original layout like in #9. It indicates that it is only a rough estimation.

It looked like: ` ~2h 17m 40s` but `~02:17:40` is also good for me.

Reply:
I like the ~, definitely. Also migrated to proposed time format, with a small adjustment: only display the two most significatives unit (i.e: minutes & seconds, hours & minutes or days & hours)

Reply:
Also, thanks for the pointer to #9. I was surprised to see there was no estimated time. Why was the feature not implemented/dropped?

Reply:
It was dropped from that pullrequest as it was very inaccurate. And we thougt about creating an issue for this later. It seems that we forgot to add it.